app/src/components/Shared/ToolTip.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import ToolTipStyle from '../../../styles/components/c-tooltip-info.scss';
+import ImgWhite from '../../../assets/icons/info_white.svg';
+import ImgBlack from '../../../assets/icons/info_black.svg';
+
+class ToolTip extends Component {
+
+  render() {
+    return (
+      <Link
+        to="/definitions"
```

Comment:
I think that behavior has nothing to do with tooltip implementation. I know in Pivotal the task relates both things, but please, separate them. There's no connection between them. That behavior will develop in other branch, not here.